From d9747b56cdad135dfcda13b707406ba43de958f3 Mon Sep 17 00:00:00 2001
From: Liam Fries <contributor@spacefox.invalid>
Date: Mon, 31 Aug 2026 18:52:20 -0400
Subject: [PATCH] Import Spacefox main files into browser.xhtml

---
 browser/base/content/browser.xhtml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/browser/base/content/browser.xhtml b/browser/base/content/browser.xhtml
index b450a27a75..ba0c75360c 100644
--- a/browser/base/content/browser.xhtml
+++ b/browser/base/content/browser.xhtml
@@ -108,6 +108,10 @@
 
   <script src="chrome://browser/content/global-scripts.js"></script>
   <script src="chrome://browser/content/browser-main.js"></script>
+
+  <!-- Spacefox chrome imports -->
+  <link rel="stylesheet" href="chrome://spacefox/content/main.css" />
+  <script src="chrome://spacefox/content/main.js"></script>
 </head>
 <html:body xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
 # All sets except for popupsets (commands, keys, and stringbundles)
-- 
2.55.0

